libs/langchain/tests/integration_tests/test_audio.py

```diff
@@ -0,0 +1,49 @@
+from langchain.document_loaders.audio import AzureSpeechServiceLoader
+
+SPEECH_SERVICE_REGION = ""
+SPEECH_SERVICE_KEY = ""
+
+
+def _get_csv_file_path() -> str:
+    return ""
+
+
+def test_azure_speech_load_key_region_auto_detect_languages() -> None:
+    loader = AzureSpeechServiceLoader(
```

Comment:
`loader = AzureSpeechServiceLoader.from_filesystem(...)`

Reply:
Dear @eyurtsev ,

AzureSpeechServiceLoader doesn't inherit the `GenericLoader` but the `BaseLoader ` just like the CSVLoader and the PDFLoader. In this case, it does not use the from_filesystem() function but load the file with class initializer like: `AzureSpeechServiceLoader(file_path: str)` format.

I uploaded the unit tests success screenshot in previous conversation. I am wondering if I upload my unit test result with console result with console and the test code (without my credential), will help your code review?

Reply:
Dear @eyurtsev ,

First appreciate you soooo much for your updated my parser neat and well-structured, I found it until I got home. I updated my loader after I got home:
1. the class `AzureSpeechServiceParser` `__init__` function kwargs' type
2. changed the loader from a GenericLoader  to BaseLoader, and added the lazy_load function
3. updated the unit tests
![Screenshot 2023-12-06 at 20 04 05](https://github.com/langchain-ai/langchain/assets/18718740/4670baf0-cd5c-4ad6-aae1-8e7b7af7019b)